soh/soh/Enhancements/randomizer/randomizerTypes.h

```diff
@@ -964,6 +964,12 @@ typedef enum {
     RG_MAX
 } RandomizerGet;
 
+typedef struct {
+    RandomizerGet rgID;
+    RandomizerGet fakeRgID;
+    char trickName[50];
+} RandomizerGetData;
```

Comment:
I already dont _love_ the name of the type RandomizerGet, and this not loving this either as it sounds more like a function name. I don’t have any better suggestions currently but I’ll think about it

Reply:
I could do `RandomizerGetObject`

Reply:
It's more about the `RandomizerGet` part than object vs data. For instance we have a handful of methods that are named very similarly but just have an underscore spacing `Randomizer_GetSettingValue` `Randomizer_GetSettingValue`, so something named `RandomizerGetObject` makes me think this is going to return a randomizer object, not an object specific to a `RandomizerGet`.

I'm not set on this blocking the PR, but just something in my head reading through this

Reply:
I see what you mean but that change would be a Herculean effort at this point.

Reply:
yeah, it'd be a giant cleanup that would need to be it's own PR (might be worth making an issue for it though)

Reply:
RandomizerGet is supposed to be for the item you get. If we're renaming that RandomizerItem makes the most sense I think. Then this can be called RandomizerItemData or RandomizeItemObject. Or even call this RandomizerItem and rename RandomizerGet to RandomizerItemID.

Reply:
i like the idea of RandomizerGet having ID in the name in some way shape or form so we don't need to add Data or something to the end of the structs, RandomizerItemID works for me but i'm open to other suggestions

regardless, that will be a large change and should probably be it's own PR